### PR TITLE
Fix SIT standalone build

### PR DESCRIPTION
**Pull request by Maxim-Doronin**
### Details:
 - Fix sing-image-test and compile_tool standalone build when OpenVINO project and ov_add_target is not reachable 

### Tickets:
 - E-132977

### CODE REVIEW

File: src/plugins/intel_npu/tools/single-image-test/cmake/standalone.cmake [hunk 1]
```diff
@@ -37,21 +37,28 @@ set(DEPENDENCIES
...
+        "If OpenVINO was built with TBB, you'll likely get a linking error. "
+        "Make sure you have called setupvars or have specified TBB_DIR.")
 endif()
 
 file(GLOB SOURCES "${CMAKE_CURRENT_SOURCE_DIR}/*.cpp")
 
 add_executable(${TARGET_NAME} ${SOURCES})
 target_link_libraries(${TARGET_NAME} PRIVATE ${DEPENDENCIES})
+if (CMAKE_COMPILER_IS_GNUCXX)
+    target_compile_options(${TARGET_NAME} PRIVATE -Wall)
```
Comment (ilya-lavrenov) [inline src/plugins/intel_npu/tools/single-image-test/cmake/standalone.cmake:60]: it's set by OpenVINO itself https://github.com/openvinotoolkit/openvino/blob/master/cmake/developer_package/compile_flags/os_flags.cmake#L447

Comment (Maxim-Doronin) [reply src/plugins/intel_npu/tools/single-image-test/cmake/standalone.cmake:60]: @ilya-lavrenov thanks for the comment!
Will this option be propagated if build this tool in standalone mode?
i.e.
find_package(OpenVINO REQUIRED COMPONENTS Runtime)
link with openvino::runtime

Comment (ilya-lavrenov) [reply src/plugins/intel_npu/tools/single-image-test/cmake/standalone.cmake:60]: hm, no
only with developer package

Comment (Maxim-Doronin) [reply src/plugins/intel_npu/tools/single-image-test/cmake/standalone.cmake:60]: Okay, I've addressed this comment where applicable https://github.com/openvinotoolkit/openvino/pull/25752/commits/f370e333bfe98a47e4e71cf233fb188cbdb21b71

**Review by pereanub — APPROVED** [review]

**Pull request merged**